modules/serverless-logic-restarting-the-build-preview-mode.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-restarting-the-build-preview-mode_{context}"]
+= Restarting a build
+
+To restart a build, you can add or edit the `sonataflow.org/restartBuild: true` annotation in the `SonataFlowBuild` instance. 
```

Comment:
Rather than previewing the details, might be good to say why a user would want/need to restart a build

Reply:
Added